x-pack/plugins/observability/public/config/paths.ts

```diff
@@ -14,10 +14,8 @@ export const paths = {
     rules: RULES_PAGE_LINK,
     ruleDetails: (ruleId?: string | null) =>
       ruleId ? `${RULES_PAGE_LINK}/${encodeURI(ruleId)}` : RULES_PAGE_LINK,
-    alertDetails: (alertId?: string | null, ruleId?: string | null) =>
-      alertId && ruleId
-        ? `${ALERT_PAGE_LINK}/rules/${encodeURI(ruleId)}/alerts/${encodeURI(alertId)}`
-        : ALERT_PAGE_LINK,
+    alertDetails: (alertId?: string | null) =>
```

Comment:
@fkanout BTW, I was thinking about the parameters. Actually they are not optional. You would still need to pass arguments while calling the function, but it could accept undefined or null. For optional arguments, you could call function without passing anything at all.

Reply:
That's great you found the API that gives us all we need about an alert without `ruleId`. 👏🏻 
Regarding the function arguments, I don't see a use case where we call this function without the `alertId`. For me, 
 the function signature should be `alertDetails: (alertId: string) =>`. Am I missing something here?

Reply:
No, you are right, I have already changed it to `alertDetails: (alertId: string) =>`.